vulcanai/net.py

```diff
@@ -434,7 +441,8 @@ def create_trainer(self):
             if self.num_classes is None or self.num_classes == 0:
                 self.cost = self.mse_loss(out, self.y)
             else:
-                self.cost = self.cross_entropy_loss(out, self.y)
+                # self.cost = self.cross_entropy_loss(out, self.y)
+                self.cost = self.sensitive_cost(out, self.y)
```

Comment:
this will do for now but once we integrate custom cost functions in the new Vulcan this will need to change